stackdriver.go

```diff
@@ -362,6 +370,12 @@ func NewExporter(o Options) (*Exporter, error) {
 	if o.MetricPrefix != "" && !strings.HasSuffix(o.MetricPrefix, "/") {
 		o.MetricPrefix = o.MetricPrefix + "/"
 	}
+	if o.UserAgent == "" {
+		o.UserAgent = defaultUserAgent
+	} else {
+		o.UserAgent = strings.Replace(o.UserAgent, "{{sdk-version}}", opencensus.Version(), -1)
```

Comment:
Is there actual use case where we need to keep both versions just with a custom user agent format?

In case of OpenTelemetry Collector we need to set it to something like`opentelemetry-collector $1; stackdriver-exporter $2`, where `$1` is the version of `opentelemetry-collector` (core), and $2 is the version of `opentelemetry-collector-contrib` (or its submodule, which are currently the same).

Reply:
Removed placeholder replacement for now